src/index.js

```diff
@@ -51,14 +51,18 @@ export default class extends three.Sprite {
     const ctx = canvas.getContext('2d');
   
     const lines = this._text.split('\n');
-
+    const font = `${this.fontWeight} ${this.fontSize}px ${this.fontFace}`;
+  
+    //Measure actual font.
```

Comment:
Could you please remove this comment line?

Reply:
# This first font setting was removed during multiline implementation, thus causing wrong width measures.
# When I've started debugging I've naturally moved font from after canvas resizing to this position. Text was all there, but very small. It took some time to understand resetting.

Given these, I think both comments should be kept in place because they specifying not very obvious things (=there are people unaware why the first or the second setting are needed).

Reply:
Ok. I've just made some small adjustments. Thanks again for the PR. 👍

Reply:
Thank you for this library and the 3d-force-graph as a whole!